libcoveocds/cli/__main__.py

```diff
@@ -1,32 +1,83 @@
-import argparse
 import json
+import os
 import shutil
+import sys
 import tempfile
 
+import click
+
 import libcoveocds.api
+from libcoveocds.config import LibCoveOCDSConfig
+
+
+@click.group()
+def cli():
+    pass
+
 
+@cli.command()
+@click.argument('filename')
+@click.option('-c', '--convert', is_flag=True, help='Convert data from nested (json) to flat format (spreadsheet)')
```

Comment:
Does this also do spreadsheet->json?

Reply:
No, it uses ocds_json_output - like old version - so it expects JSON